webapp/loggedin/countrySelection/components/countryList.js

```diff
@@ -0,0 +1,77 @@
+import './countryList.less'
+
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import * as R from 'ramda'
+
+import { isAdministrator } from '@common/countryRole'
+import { getRelativeDate } from '@webapp/utils/relativeDate'
+
+import { CSVLink } from 'react-csv'
+import Icon from '@webapp/components/icon'
+import CountryListRoleSection from '@webapp/loggedin/countrySelection/components/countryListRoleSection'
+import useUserInfo from '@webapp/components/hooks/useUserInfo'
+import useI18n from '@webapp/components/hooks/useI18n'
+
+import * as CountryState from '@webapp/country/countryState'
+
+import { getCountryName } from '@webapp/country/actions'
+
+const CountryList = () => {
+  const dispatch = useDispatch()
+  const userInfo = useUserInfo()
+  const i18n = useI18n()
+  const countries = useSelector(CountryState.getCountries)
+  const roleCountriesPair = R.toPairs(countries)
+
+  return (
+    <div className="country-selection-list">
+      {
+        isAdministrator(userInfo) &&
+        <div className="country-selection-list-download">
+          <CSVLink
+            className="btn-s btn-primary"
+            target="_blank"
+            filename="FRA-Countries.csv"
+            data={
```

Comment:
Should data prop value be defined somewhere else for readability?

Reply:
yes. I have also extracted it to a separate component